src/utils/useApolloQuery.js

```diff
@@ -0,0 +1,81 @@
+import {getApolloContext} from '@apollo/react-common';
+import {QueryData} from '@apollo/react-hooks/lib/data/QueryData';
+import {useDeepMemo} from '@apollo/react-hooks/lib/utils/useDeepMemo';
+import {useContext, useEffect, useReducer} from 'react';
+
+// TODO: might be better in a context
+const queryDataRefs = new Map();
+
+// taken from https://github.com/apollographql/react-apollo/blob/master/packages/hooks/src/utils/useBaseQuery.ts
+export default function useBaseQuery(query, options) {
+	const context = useContext(getApolloContext());
+	const [tick, forceUpdate] = useReducer(x => x + 1, 0);
+	const updatedOptions = options
+		? Object.assign(Object.assign({}, options), {query})
```

Comment:
```suggestion
		? {...options, {query: ...query}}
```
ca revient au même si je me trompe pas.

Reply:
```suggestion
		? {...options, query}
```
Si la ref a la query peut être gardé.